Cargo.toml

```diff
@@ -11,15 +11,22 @@ license = "MIT/Apache-2.0"
 keywords = ["http", "reader", "buffer"]
 
 [features]
-default = ["reqwest"]
+default = ["reqwest-sync", "reqwest-async"]
+logging = ["log"]
```

Comment:
More typically, logging output is handled by configuring LOG_LEVEL, e.g. `LOG_LEVEL=debug,http-range-client-debug=trace`. 

If you are worried about bloat from the log messages, the end user can compile out certain levels altogether via: https://docs.rs/log/latest/log/index.html#compile-time-filters

I appreciate that `log` is so ubiquitous that I don't need to look up documentation for every crate to know how to enable it's logging. Is there a reason to diverge?

Reply:
We do a little bit more than logging by aggregating some read statstics, which is not necessary in some cases. But I'm fine with removing ths feature flag, since reqwest depends on the logger anyway.